server/db/data/cassandraAssociated.cql

```diff
@@ -0,0 +1,16 @@
+CREATE KEYSPACE IF NOT EXISTS videoplayer WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
```

Comment:
Since we are getting ready for deployment I suggest not using the SimpleStrategy. Use NetworkTopology instead.